server/src/modules/topics/topics.routes.ts

```diff
@@ -0,0 +1,80 @@
+import { AuthMiddleware } from '../auth/auth.middleware'
+import express from 'express'
+import { check, param } from 'express-validator'
+import { TopicsController } from './topics.controller'
+
+export const routeTopics = ({
+  controller,
+  authMiddleware,
+}: {
+  controller: TopicsController
+  authMiddleware: Public<AuthMiddleware>
+}): express.Router => {
+  const router = express.Router()
+
+  /**
+   * Lists all topics corresponding to an agency
+   * @route   GET /api/topics/agency/:agencyId
+   * @returns 200 with list of nested topics
+   * @returns 400 with database error
+   */
+  router.get(
+    '/:agencyId',
```

Comment:
```suggestion
    '/agency/:agencyId',
```
To line up with the jsdoc

Reply:
A clearer alternative may be to have a path like `/agencies/:agencyId/topics`, but that would require you to modify the agency route, injecting it with the topics controller

Reply:
went with the latter suggestion - changed the route to `/agencies/:agencyId/topics`